scipy/io/wavfile.py

```diff
@@ -499,52 +531,70 @@ def read(filename, mmap=False):
     """
     Open a WAV file.
 
-    Return the sample rate (in samples/sec) and data from a WAV file.
+    Return the sample rate (in samples/sec) and data from an LPCM WAV file.
 
     Parameters
     ----------
     filename : string or open file handle
-        Input wav file.
+        Input WAV file.
     mmap : bool, optional
-        Whether to read data as memory-mapped.
-        Only to be used on real files (Default: False).
+        Whether to read data as memory-mapped (default: False).  Not compatible
+        with some bit depths; see Notes.  Only to be used on real files.
 
         .. versionadded:: 0.12.0
 
     Returns
     -------
     rate : int
-        Sample rate of wav file.
+        Sample rate of WAV file.
     data : numpy array
-        Data read from wav file. Data-type is determined from the file;
+        Data read from WAV file. Data-type is determined from the file;
         see Notes.  Data is 1-D for 1-channel WAV, or 2-D of shape
         (Nsamples, Nchannels) otherwise. If a file-like input without a
         C-like file descriptor (e.g., :class:`python:io.BytesIO`) is
         passed, this will not be writeable.
 
     Notes
     -----
-    This function cannot read wav files with 24-bit data.
-
     Common data types: [1]_
 
     =====================  ===========  ===========  =============
          WAV format            Min          Max       NumPy dtype
     =====================  ===========  ===========  =============
     32-bit floating-point  -1.0         +1.0         float32
-    32-bit PCM             -2147483648  +2147483647  int32
-    16-bit PCM             -32768       +32767       int16
-    8-bit PCM              0            255          uint8
+    32-bit integer PCM     -2147483648  +2147483647  int32
```

Comment:
Is 64-bit floating-point supported? L442 makes me suspect yes

Reply:
Yes but I don't think any such files exist in the real world.  They certainly aren't common

Reply:
ahh, I read that comment in another place but didn't put 2 and 2 together.